src/kinetics/Reaction.cpp

```diff
@@ -359,30 +359,28 @@ void readFalloff(FalloffReaction& R, const XML_Node& rc_node)
         falloff_parameters.push_back(fpValueCheck(p[n]));
     }
 
-    int falloff_type = 0;
     if (caseInsensitiveEquals(falloff["type"], "lindemann")) {
-        falloff_type = SIMPLE_FALLOFF;
         if (np != 0) {
             throw CanteraError("readFalloff", "Lindemann parameterization "
                 "takes no parameters, but {} were given", np);
         }
+        R.falloff = newFalloff("Simple", falloff_parameters);
```

Comment:
Does it make sense to use `Lindemann` as the name `Simple`, which follows more naturally from a reading of the [science docs](https://cantera.org/science/reactions.html#falloff-reactions)?

Reply:
I will change the default name to `Lindemann` and set up a synonym for `Simple`.